easybuild/easyconfigs/n/numactl/numactl-2.0.11-GCCcore-7.1.0.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'ConfigureMake'
+
+name = 'numactl'
+version = '2.0.11'
+
+homepage = 'http://oss.sgi.com/projects/libnuma/'
+description = """The numactl program allows you to run your application program on specific cpu's and memory nodes.
+ It does this by supplying a NUMA memory policy to the operating system before running your program.
+ The libnuma library provides convenient ways for you to add NUMA memory policies into your own program."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.1.0'}
```

Comment:
Just so you know: I'm about to open a pull request for the just released GCC 7.2.0 together with binutils 2.29...